client/src/pages/Homepage.js

```diff
@@ -72,8 +72,9 @@ const Homepage = () => {
                 savedMovies: data.me.savedMovies
             })
 
-            data.me.removedMovies.forEach((movieId) => {
-                idbPromise('removedMovies', 'put', { movieId });
+            data.me.removedMovies.forEach((movie) => {
+                console.log("movieeeee", movie)
+                idbPromise('removedMovies', 'put', { movieId: movie.movieId });
```

Comment:
should removedMovies put movie instead of movie.movieId? If we're defining removedMovies as an array of movieInputs, we should just make it standard across all of our stored data, esp if the intent is to include the ability to see removed movies.